tests/run-make/rustdoc-io-error/rmake.rs

```diff
@@ -20,7 +20,7 @@ use run_make_support::{path, rustdoc};
 
 fn main() {
     let out_dir = path("rustdoc-io-error");
-    let output = fs::create_dir(&out_dir).unwrap();
+    fs::create_dir(&out_dir).unwrap();
```

Comment:
Suggestion: import `run_make_support::rfs` (`std::fs` wrappers which panic on error and shows paths on failure).

```suggestion
    rfs::create_dir(&out_dir);
```

Reply:
Done.

In this case it may have been nice if it was named `run_make_support::fs`, we would only have had to change the import and remove the unwraps, instead of also changing every call site.

Reply:
It's named internally as `fs` but re-export as `rfs` since we didn't want to collide with `use std::fs;`, but yeah probably we should've just export `fs` as-is. I don't have a strong opinion on which one to pick, open to renaming it back to `fs` once we deal with the Makefiles first.